apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -32,19 +30,16 @@ const Body = ({
   };
   onClose: () => void;
 }) => {
-  const { onResult, onCancel } = data;
-  const { fromAccount: account, fromParentAccount: parentAccount } = data.exchange;
+  const { onResult, onCancel, ...exchangeParams } = data;
+
+  const { fromAccount: account, fromParentAccount: parentAccount } = exchangeParams.exchange;
   let tokenCurrency;
   if (account.type === "TokenAccount") tokenCurrency = account.token;
   const request = {
-    ...data,
-    exchange: toExchangeRaw(data.exchange),
-    transaction: toTransactionRaw(data.transaction),
```

Comment:
We don't need the raw anymore ?

Reply:
That was where the regression came from. I suppose since we don't have multi-process logic anymore serialisation is not needed. Probably a miss

Reply:
We probably miss that in other parts of the code